WordPress/gutenberg
# Move widget-area to edit-widgets

## Description


Move `widget-area` block to `edit-widgets`.

## How has this been tested?



1. `rm -rf build`
2. `npm run dev`
3. Go to widgets screen
4. Confirm it works as expected

## Types of changes




Bug fix

## Checklist:
